CHANGELOG.md

```diff
@@ -18,7 +18,7 @@ Status: Available for use
 ### Fixed
 
 - Fix up clippy warnings and enforce clippy going forward
-
+- Change docker in docker image tag from `stable-dind` to `dind`
```

Comment:
Technically this isn't a fix (this isn't broken) so this probably belongs under a Changed title.

If you also fix the `image_exists_locally` procedure, please also add a note here about that.